python/gtsam/utils/plot.py

```diff
@@ -125,16 +138,15 @@ def plot_point2_on_axes(axes,
     if P is not None:
         w, v = np.linalg.eig(P)
 
-        # "Sigma" value for drawing the uncertainty ellipse. 5 sigma corresponds
-        # to a 99.9999% confidence, i.e. assuming the estimation has been
-        # computed properly, there is a 99.999% chance that the true position
-        # of the point will lie within the uncertainty ellipse.
-        k = 5.0
+        # Sigma value corresponding to the covariance ellipse
+        k = 5
 
         angle = np.arctan2(v[1, 0], v[0, 0])
+        # We multiply k by 2 since k corresponds to the radius but Ellipse uses
```

Comment:
Wow. I guess we were looking at the wrong ellipses all that time!

Reply:
On the bright side, your numbers would have still been correct!

The requested changes have been made. Please let me know if you need anything else.

Thanks!